tests/target/single-line-simple-if.rs

```diff
@@ -0,0 +1,198 @@
+// rustfmt-single_line_simple_if: true
+// rustfmt-unstable_features: true
+
+fn main() {
+    // if statements may be formatted on a single line if they are "short"
+    // and only contain a single expression of 'return', 'continue' or 'break'
+    if true { continue }
+
+    if true { continue }
+
+    // Default max width is 50
+    if width == 50_characters_or_shorter { continue }
+    if width == 51_characters_long_and_above {
+        return;
+    }
+
+    if name == super_duper_really_really_mega_ultra_giga_long_name_with_a_cherry_on_top {
+        return;
+    }
+
+    // More than 1 stmt means a new line, it is no longer 'simple'
+    if a {
+        let y = 1;
+        return y;
+    }
+
+    // Adds a semicolon to 'return/continue/break' when put on a new line
+    // (unless config has trailing_semicolon = false)
+    if a {
+        let y = 1;
+        return y;
+    }
+
+    // Will not work on fn or method calls (may change)
+    if true {
+        do_something()
+    }
+
+    // Will not work on an expression with trailing semicolon pre-format
+    if true {
+        return;
+    }
+
+    // Will not single line if there is an else block, even with single expressions
+    if true {
+        return;
+    } else {
+        break;
+    }
+
+    // Will not be single line if returns/breaks with a value
+    for i in 0..2 {
+        if true { break }
+        if true {
+            break 2;
+        }
+        if true { return }
+        if true {
+            return 3;
+        }
+    }
+
+    // Will not be single line if comment is in the block
+    if true {
+        // nope
+        return;
+    }
+    if true {
+        /* nope 2 */
+        return;
+    }
+
+    // Only works on if blocks, not other control flow
+    for i in 0..2 {
+        if i == 1 { continue }
+    }
+
+    for i in 0..2 {
+        loop {
+            if i == 1 { continue }
+        }
+    }
+
+    // New line formatted here as 'loop' != 'return/continue/break'
+    if i == 1 {
+        loop {
+            return;
+        }
+    }
+
+    // Works on labelled break/continue
+    'gamer: loop {
+        if true { break 'gamer }
```

Comment:
Can you add a test case for `continue` here too?

Reply:
Should I also put in examples of it working with `if let`?